app/styles/style-flat.sass

```diff
@@ -167,20 +176,20 @@ $forest: #20572B
     border: 1px solid $navy
     color: $navy
     
-  .btn-forest
+  .btn-secondary, .btn-forest
     background-color: $forest
     color: white
     
-  .btn-forest-alt
+  .btn-secondary-alt, .btn-forest-alt
     background-color: white
     border: 1px solid $forest
     color: $forest
     
-  .btn-gold
+  .btn-tertiary, .btn-gold
     background-color: $gold
     color: white
     
-  .btn-gold-alt
+  .btn-tertiary-alt, .btn-gold-alt
```

Comment:
Let's get rid of this layer of abstraction. If I see .btn-tertiary-alt, then I have to remember what the tertiary color is.